Gemfile

```diff
@@ -50,12 +50,18 @@ gem "bootsnap", require: false
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
-  gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'factory_bot_rails'
+  gem 'faker'
+  gem 'rails-controller-testing'
+  gem 'rspec-rails'
+  gem 'rubocop', '>= 1.0', '< 2.0'
 end
```

Comment:
- [ ] I suggest that you use the Capybara testing library for the testing of your views :+1: 
  
  ```ruby
   #Gemfile
  
   group :development, :test do
     ...
     gem 'capybara'
     gem 'rspec-rails'
   end
  ```

Reply:
I'm using it! Line 75 in the Gemfile :)